WasatchNET/MockSpectrometer.cs

```diff
@@ -314,6 +314,107 @@ public override bool reconnect()
             return true;
         }
 
+        public override double[] getSpectrum(bool forceNew = false)

```

Comment:
I guess something I'd recommend down the road is that anytime you implement a non-trivial override of a base class function, include a brief comment explaining why you needed to do the override or how it differs from the original (so they don't have to do a line-by-line diff to see what you changed).

Reply:
good call